package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
```

Comment:
a little bit off topic but I've been meaning to ask, can we get rid of Bootstrap on this project?

I think it kind makes things a lil confusing because they have specific classes that you if you can call on by accident and also if something is not working properly its hard to tell if its your fault or because maybe someone else used Bootstrap class somewhere else

Also some people might not be aware it exists so if they try to make a class called "container" and they didn't want to have flexbox behaviour, well they might be confused because by default "container" in Bootstrap have flexbox behaviour


Summary: can we either get rid of Bootstrap or can we make a note somewhere that it's being used so people in the future will know to be careful?

Reply:
I think we should put it in the developer tag at the top in Key Options:

https://proudtale.atlassian.net/wiki/spaces/PP/blog/2020/05/23/132907009/How+to+properly+tag+your+js+files

/**
 * Copyright (c) 2020 proudtale
 * All rights reserved.
 *
 * Filename: tagging.js
 *
 * Key Options:
 * - set a valid value for [this booktitle]
 * - Boostrap is used with the flexbox behaviour in this js file.
 *
 * Revision History:
 * - 22 MAR 2020, Marko Tanuan <markotanuan@proudtale.ca> : Created
 * - 17 MAY 2020, Sidney Wittman <sidney.wittman@proudtale.com> : Refactor to a separate js file
 *  
 */

Reply:
> a little bit off topic but I've been meaning to ask, can we get rid of Bootstrap on this project?
> 
> I think it kind makes things a lil confusing because they have specific classes that you if you can call on by accident and also if something is not working properly its hard to tell if its your fault or because maybe someone else used Bootstrap class somewhere else
> 
> Also some people might not be aware it exists so if they try to make a class called "container" and they didn't want to have flexbox behaviour, well they might be confused because by default "container" in Bootstrap have flexbox behaviour
> 
> Summary: can we either get rid of Bootstrap or can we make a note somewhere that it's being used so people in the future will know to be careful?

I will check which tag is using bootstrap and I will remove and use material UI.

Reply:
> > a little bit off topic but I've been meaning to ask, can we get rid of Bootstrap on this project?
> > I think it kind makes things a lil confusing because they have specific classes that you if you can call on by accident and also if something is not working properly its hard to tell if its your fault or because maybe someone else used Bootstrap class somewhere else
> > Also some people might not be aware it exists so if they try to make a class called "container" and they didn't want to have flexbox behaviour, well they might be confused because by default "container" in Bootstrap have flexbox behaviour
> > Summary: can we either get rid of Bootstrap or can we make a note somewhere that it's being used so people in the future will know to be careful?
> 
> I will check which tag is using bootstrap and I will remove and use material UI.

I used bootstrap for header and footer mainly. I will replace it with Material UI.